src/components/AddressDetailExplorer.js

```diff
@@ -167,26 +175,24 @@ class AddressDetailExplorer extends React.Component {
    *
    * @param {Object} queryParams URL parameters
    */
-  getHistoryData = (page) => {
-    // update query parameters
-    const query = this.pagination.obtainQueryParams();
-    if (query.page !== page) {
-      query.page = page;
-      const newURL = this.pagination.setURLParameters(query);
-      this.props.history.push(newURL);
-    }
-
-    return addressApi.getHistory(this.state.address, this.state.selectedToken, TX_COUNT, TX_COUNT * (page-1)).then((response) => {
+  getHistoryData = (lastTx, lastTs) => {
+    console.log('Downloading')
+    return addressApi.getHistory(this.state.address, this.state.selectedToken, TX_COUNT, lastTx, lastTs).then((response) => {
       if (!response) {
         // An error happened with the API call
-        this.setState({ showReloadTokenButton: true });
+        this.setState({
+          showReloadTokenButton: true,
```

Comment:
It seems you used a linter here. Is this linter configured in the project to check all the PRs? Or is it from your IDE?

We should make sure we have a default style and linters that check it on PRs, to avoid everyone using their own linters.

Reply:
I actually manually changed it, we don't have a default style configured for this project yet